### PR TITLE
add recipe for aurora

### PR DESCRIPTION
**Pull request by lheagy**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/aurora/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+about:
+  home: https://github.com/simpeg/aurora
+  summary: Processing Codes for Magnetotelluric Data
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - kkappler
```
Comment (synapticarbors) [inline recipes/aurora/meta.yaml:50]: Can you verify either that you have spoken to the listed co-maintainers offline or have them comment on this PR that they are ok with being listed?

Comment (kujaku11) [reply recipes/aurora/meta.yaml:50]: I'm good with changes.

Comment (kkappler) [reply recipes/aurora/meta.yaml:50]: Hi, Karl here - yes I'm happy to be a maintainer!

**Comment by lheagy** [comment]
Thanks @synapticarbors! Yes, both @kujaku11 and @kkappler have confirmed

**Pull request merged**